social_core/tests/backends/test_saml.py

```diff
@@ -26,8 +29,8 @@
 DATA_DIR = path.join(path.dirname(__file__), 'data')
 
 
-@unittest2.skipIf('TRAVIS' in os.environ and sys.version_info[:2] == (3, 5),
-                  'Travis-ci segfaults in Python 3.5 probably due to a bad ' \
+@unittest2.skipIf('TRAVIS' in os.environ,
```

Comment:
Strange thing, [those tests](https://github.com/python-social-auth/social-core/pull/13/commits/55a03b65df64dd6a9e62f2af49b24e0c0b530022#diff-5bf0a2e8ccd1bff565f8367d724e1d11R72) passed in my PR https://travis-ci.org/python-social-auth/social-core/jobs/188914770